identity-service/default-config.json

```diff
@@ -29,6 +29,7 @@
   "rateLimitingListensPerTrackPerWeek": 120,
   "rateLimitingListensPerIPPerWeek": 180,
   "rateLimitingListensIPWhitelist": "(.*192.168.*|ffff:172.*)",
+  "endpointRateLimits": "{\"/artist_pick\":{\"post\":[{\"expiry\":60,\"max\":40}]},\"/user\":{\"post\":[{\"expiry\":60,\"max\":100}]},\"/users/update\":{\"post\":[{\"expiry\":60,\"max\":100}]},\"/user_playlist_favorites\":{\"post\":[{\"expiry\":60,\"max\":40}]},\"/recovery\":{\"post\":[{\"expiry\":60,\"max\":5}]},\"/email/welcome\":{\"post\":[{\"expiry\":60,\"max\":5}]},\"/social_handles\":{\"post\":[{\"expiry\":60,\"max\":40}]},\"/push_notifications/settings\":{\"post\":[{\"expiry\":60,\"max\":40}]},\"/push_notifications/device_token\":{\"post\":[{\"expiry\":60,\"max\":40}]},\"/push_notifications/device_token/deregister\":{\"post\":[{\"expiry\":60,\"max\":40}]},\"/push_notifications/browser/settings\":{\"post\":[{\"expiry\":60,\"max\":40}]},\"/push_notifications/browser/register\":{\"post\":[{\"expiry\":60,\"max\":40}]},\"/push_notifications/browser/deregister\":{\"post\":[{\"expiry\":60,\"max\":40}]},\"/userEvents\":{\"post\":[{\"expiry\":60,\"max\":100}]},\"/twitter\":{\"post\":[{\"expiry\":60,\"max\":40}]},\"/twitter/callback\":{\"post\":[{\"expiry\":60,\"max\":40}]},\"/twitter/associate\":{\"post\":[{\"expiry\":60,\"max\":40}]},\"/instagram\":{\"post\":[{\"expiry\":60,\"max\":40}]},\"/instagram/associate\":{\"post\":[{\"expiry\":60,\"max\":40}]},\"/notifications\":{\"post\":[{\"expiry\":60,\"max\":40}]},\"/notifications/all\":{\"post\":[{\"expiry\":60,\"max\":40}]},\"/notifications/clear_badges\":{\"post\":[{\"expiry\":60,\"max\":40}]},\"/notifications/settings\":{\"post\":[{\"expiry\":60,\"max\":40}]},\"/announcements\":{\"post\":[{\"expiry\":60,\"max\":40}]},\"/notifications/subscription\":{\"post\":[{\"expiry\":60,\"max\":40}]},\"/authentication\":{\"post\":[{\"expiry\":60,\"max\":40}]}}",
```

Comment:
i see you've added an authentication POST limit. there's already an existing GET limit for that. should we migrate that to this format?

Reply:
I didn't want to change any of the existing limits. figured we could just do that after if that's ok w/ u?

Reply:
definitely